File: scala-gatling/src/gatling/scala/org/openapitools/client/model/RateLimitedError.scala

package org.openapitools.client.model


case class RateLimitedError (
    _result: AnyType,
    _msg: AnyType,
    _code: Option[AnyType]
)
object RateLimitedError {
    def toStringBody(var_result: Object, var_msg: Object, var_code: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg,"code":$var_code
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/RateLimitedError.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.AnyType
open OpenAPI.Model.CodedError

module RateLimitedError = 

  //#region RateLimitedError

  [<CLIMutable>]
  type RateLimitedError = {
    [<JsonProperty(PropertyName = "result")>]
    Result : AnyType;
    [<JsonProperty(PropertyName = "msg")>]
    Msg : AnyType;
    [<JsonProperty(PropertyName = "code")>]
    Code : AnyType;
  }
  
  //#endregion
  